docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+| `BC_APP_CLIENT_ID` | The app API account's client ID | [View App Credentials](/api-docs/apps/guide/developer-portal#view-credentials?source=subscription-foundation) |
+| `BC_APP_SECRET` | The app API account's client secret | [View App Credentials](/api-docs/apps/guide/developer-portal#view-credentials?source=subscription-foundation) |
+| `NEXT_PUBLIC_STRIPE_CLIENT_ID` | The app-specific Stripe Connect API account client ID | see [Enable Stripe Connect for Platforms](#enable-stripe-connect-for-platforms) |
+| `STRIPE_SECRET_KEY` | The app-specific Stripe Connect API account client secret | see [Create Stripe Accounts](#create-stripe-accounts) |
+
+
+## Run migration and start the server
+
+<!-- theme: info -->
+> #### Database note
+> This example uses SQLite as a data store. In production, we recommend using a database that has more robust concurrency support, such as PostgreSQL. For information on switching databases, see the [Replacing SQLite](#replacing-sqlite) section.
+
+1. If you're using SQLite, which is Subscription Foundation's default data store, skip to the next step. Otherwise, see [Replacing SQLite](#replacing-sqlite)
+
+2. Run the pre-configured Prisma migration to create the database tables and initial client as defined in `/prisma/migrations/*`. Troubleshoot as needed.
+
+```shell title="Run Prisma migration"
+npx prisma migrate dev
+```
+
+![npx_prisma_migrate](https://storage.googleapis.com/bigcommerce-production-dev-center/images/npx_prisma_migrate.png)
+
+3. Start the app server.
+
+```shell title="Start the app server"
+npm run dev
+```
+
+4. Start ngrok using the follow command. If your app does not run on port 3000, replace `3000` with your app server's port.
+
+```shell title="Start ngrok"
+ngrok http 3000
+```
+
+Once the app server and ngrok are running, you can install the draft app on your sandbox store. For more about installing and troubleshooting apps in development, see our App Tutorial section on [Installing and launching an app](/api-docs/apps/tutorials/sample-app-nextjs/step-2-connect#install-and-launch-the-app).
+
+## Replacing SQLite
+
+To use an alternate SQL database, do the following:
+
+1. Update the `/prisma/schema.prisma` file with a `provider` other than `SQLite`. Consult [Prisma's reference docs](https://prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-sources/) for a list of options.
+
+2. In `/prisma/.env`, update the value of the `DATABASE_URL` variable to match your new database connection string.
+
+3. Run `npx prisma migrate dev` per the preceding section.
+
+4. Run `npx prisma generate` to generate a fresh app client that uses the new database provider.
+
+5. To access this database locally, run `npx prisma studio` and use a visual editor to verify that the tables have been created.
+
+6. Start the app server and then ngrok, per the preceding section.
```

Comment:
I would write out the commands for starting the app server and ngrok: nom run dev and ngrok http 3000